docs/dev-process/coding-guidelines.md

```diff
@@ -111,9 +111,6 @@ Some general guidelines:
 * Avoid using string interpolations and overloads with implicit parameters such as `Culture` and `StringComparison`.
   Instead, use overloads with more explicit parameters such as `String.Format(IFormatProvider, String, Object[])` and `Equals(String, String, StringComparison)`.
 
-* Avoid creating empty arrays.
```

Comment:
Unnecessary to make this guideline explicit when we can rely on the compiler to warn against this pattern.

Reply:
I think we will update the guidance after we finish our current migration to Roslyn rules.

Reply:
So we'll keep this removal?